lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -30,11 +30,11 @@ module Billing #:nodoc:
     class OrbitalGateway < Gateway
       include Empty
 
-      API_VERSION = "5.6"
+      API_VERSION = "7.1"
 
       POST_HEADERS = {
         "MIME-Version" => "1.1",
-        "Content-Type" => "application/PTI56",
+        "Content-Type" => "application/PTI#{API_VERSION.gsub(/\./, '')}",
```

Comment:
Hmm, interesting thought. But they do not refer to the schema version as `PTI71`, that's just the format it's expected in for the schema validator to pick up. This way seems marginally readable, though I suppose it's possible the gsub could break somehow...